src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -653,19 +653,28 @@ public String[] getApplicationProtocols() {
      * {@code String} should be exchanged using {@code UTF-8}, the
      * {@code String} should be converted to its {@code byte[]} representation
      * and stored as a byte-oriented {@code String} before calling this method.
+     * For example:
      *
      * <blockquote><pre>
-     *     // MEETEI MAYEK LETTERS HUK UN I (Unicode 0xabcd->0xabcf): 2 bytes
-     *     byte[] bytes = "\u005cuabcd\u005cuabce\u005cuabcf"
-     *             .getBytes(StandardCharsets.UTF_8);
-     *     String HUK_UN_I = new String(bytes, StandardCharsets.ISO_8859_1);
+     *     // Encode 3 Meetei Mayek letters (HUK, UN, I) using Unicode Escapes
+     *     //     0xabcd->0xabcf, 2 Unicode bytes/letter.
+     *     String HUK_UN_I =  "\u005cuabcd\u005cuabce\u005cuabcf";
```

Comment:
```
jshell> String HUK_UN_I =  "\u005cuabcd\u005cuabce\u005cuabcf";
|  Error:
|  illegal escape character
|  String HUK_UN_I =  "\u005cuabcd\u005cuabce\u005cuabcf";
|                            ^
|  Error:
|  illegal escape character
|  String HUK_UN_I =  "\u005cuabcd\u005cuabce\u005cuabcf";
|                                       ^
|  Error:
|  illegal escape character
|  String HUK_UN_I =  "\u005cuabcd\u005cuabce\u005cuabcf";
|                                                  ^
```

Reply:
javadoc on "\u005c" produces the "\" character.  

So "\u005cuabcd" renders "\uabcd..."

The resulting javadoc looks like:

     // Encode 3 Meetei Mayek letters (HUK, UN, I) using Unicode Escapes
     //     0xabcd-&gt;0xabcf, 2 Unicode bytes/letter.
     String HUK_UN_I =  "\uabcd\uabce\uabcf";

That was the way it was done elsewhere in the JDK (e.g. `Arrays`/`URI`/etc.) minus the `{@code...}`, so did the same.